addons/modules/functions/fnc_gui_executeCode.sqf

```diff
@@ -101,8 +101,12 @@ private _fnc_onConfirm = {
             [QEGVAR(common,execute), [_code, _args]] call CBA_fnc_globalEvent;
         };
         case 3: {
-            private _jipID = [_args, _code] remoteExec ["call", 0, _logic];
-            _logic setName format ["JIP ID: %1", _jipID];
+            private _jipID = [QEGVAR(common,execute), [_code, _args]] call CBA_fnc_globalEventJIP;
+            [_jipID, _logic] call CBA_fnc_removeGlobalEventJIP;
```

Comment:
I think I'm missing something but why create a JIP event and then right after you remove it?

Reply:
It's removed after the logic is deleted.